components/TextField/index.jsx

```diff
@@ -58,6 +62,8 @@ export default class TextField extends React.Component {
   static defaultProps = {
     autoComplete: 'off',
     autoFocus: false,
+    badwordsCallback: () => {},
```

Comment:
This works as event.
I prefer `onBadWord` or something like this.
This name is confusing, it looks like query backwords.